src/pages/plugins/getting-started/index.md

```diff
@@ -0,0 +1,115 @@
+# Building your first plugin
+
+This tutorial will walk you through the process of building your first plugin. You will learn how to create a plugin, add a command to it, and then run it in InDesign.
+
+## Prerequisites
+
+Before you start, make sure you have the following software installed on your machine:
+
+- **InDesign 2023** or later (can be installed from the _Creative Cloud Desktop_ app)
+- **UXP Developer Tool** (can be installed from the _Creative Cloud Desktop_ app)
+- A **code editor** of your choice (for example, [Visual Studio Code](https://code.visualstudio.com/))
+
+## Steps
+
+### Step 1: Create a plugin
+
+You can create your plugin directly in the UXP Developer Tool. To do so, open the UXP Developer Tool and click **Create Plugin**.
+
+This opens a dialog where you can enter the details for your plugin:
+
+<figure>
+  <img src="https://picsum.photos/1280/720" alt="Screenshot of the create plugin dialog" />
```

Comment:
is this image temporary?
I see a bunch of trees in the image

Reply:
(that's true for all images that are supposed to be screenshots)

Reply:
These are updated now.